### ISSUE TITLE
more than one device on one bridge

### ISSUE THREAD

**Issue by martinschki** [originallyus/node-red-contrib-alexa-local#29]
I know its on your todo, but do you have a timeline here?

Thanks for the awesome node!!!

**Comment by torinnguyen** [comment]
Why exactly do you need this feature? It is more of an optimization rather
than a new feature. It does not affect how you would use the node from user
point of view.

**Comment by martinschki** [comment]
Ok, let me rephrase my question. If adding lots of devices, would that have a negative impact in the systems / networks performance compared to lets say 20 devices per bridge?

**Comment by torinnguyen** [comment]
It is definitely better for system performance when 25 devices (max) are hosted on 1 bridge. However the implementation is challenging and I have not thought to a way to keep the configuration of the node as idiot-proof as it is now.
